pkgs/development/tools/parsing/tree-sitter/update_impl.py

```diff
@@ -96,7 +117,7 @@ def fetchRepo() -> None:
             "outputDir": outputDir,
             "nixRepoAttrName": nixRepoAttrName,
         }:
-            token: str | None = os.environ.get("GITHUB_TOKEN", None)
+            token: str | None = os.environ.get("NIXPKGS_GITHUB_TOKEN", None)
```

Comment:
Belongs in a different commit.

Reply:
Why this change? Seems like it would unexpectedly make for extra work for someone trying to update?

Reply:
The only occurrence is on master, where it explains how to use the script: https://github.com/NixOS/nixpkgs/blob/d1ff9abb5ce634284f78ecebbf4cb622bc866817/pkgs/development/tools/parsing/tree-sitter/default.nix#L29

The comment refers to `NIXPKGS_GITHUB_TOKEN`, while the script uses `GITHUB_TOKEN`, which is misleading. This change is necessary to make the comment useful. I’m fine with either name, but I felt prefixing it with `NIXPKGS_` was clearer.

Reply:
So that's _kind of_ my fault, I mistakenly updated a comment stating it was `GITHUB_TOKEN` to `NIXPKGS_GITHUB_TOKEN` (because I got unthrottled when I used that one). Someone reached out to let me know this code uses `GITHUB_TOKEN` (so I'm not sure why/how I found `NIXPKGS_GITHUB_TOKEN` to be hepful...).

Reply:
Yeah, we should use the comment instead, though. There's no reason to have a unique environment variable for this one updater when the entirety of nixpkgs uses the `GITHUB_TOKEN` variable.

Reply:
Thanks, amended.